src/workers/notifications/squadMemberJoined.ts

```diff
@@ -75,6 +75,7 @@ const worker: NotificationWorker = {
           post,
           source,
           doneBy,
+          initiatorId: doneBy.id,
```

Comment:
Even though admin blocked the user I think they should still see the notification if they are admin of the squad.